src/modules/note.ts

```diff
@@ -297,15 +294,23 @@ const whois = new util.RootModule(
       }
     }
 
-    // Handling for notes
-    const record: userRecord | null = await getRecord(member.user);
+    // Finally, list notes
+    const record: UserRecord | null = await getRecord(member.user);
+    let footerText;
```

Comment:
nit:
Possibly add a comment explaining the purpose of this variable?

Reply:
Which one, footerText or userRecord?